faq.md

```diff
@@ -107,9 +107,9 @@ as handwritten code. If you experience any weirdness, file a bug!
 
 <!-- References -->
 
-[`@Binds`]: http://google.github.io/dagger/api/latest/dagger/Binds.html
-[`@Provides`]: http://google.github.io/dagger/api/latest/dagger/Provides.html
-[`dagger.producers.monitoring`]: http://google.github.io/dagger/api/latest/dagger/producers/monitoring/package-summary.html
+[`@Binds`]: api/latest/dagger/Binds.html
```

Comment:
This allows testing locally with links to to-be-introduced APIs, but I could revert those changes (and "absolutize" the few existing relative references).

If we go that way, can we move the links to HTTPS?